File: lesson2/lesson2.cpp
#include <iostream>

//
// A template class describing a 2d point
//
#include "point.h"

//
// ANSI escape codes
//
#include "ansi.h"

//
// Terminal info
//
#include "terminal.h"

int main (int argc, char *argv[])
{
    Terminal term;

    std::cout << term.get_code(term.FOREGROUND_GREEN);
    std::cout << "C++: Your terminal size is " << term.size;
    std::cout << term.get_code(term.RESET);

    return (0);
}

File: lesson2/lesson2.py
'''
lesson2 - get the terminal size
'''

from terminal import Terminal
import sys


def lesson2():
    """ term size """
    term = Terminal()

    sys.stdout.write(term.get_code(term.Code.FOREGROUND_GREEN))
    sys.stdout.write("Python: Your terminal size is {}".format(term.size))
    sys.stdout.write(term.get_code(term.Code.RESET))

lesson2()
